package.json

```diff
@@ -18,11 +18,7 @@
     "typescript"
   ],
   "main": "index.js",
-  "browser": {
-    "protobufjs": "./index.js",
-    "protobufjs/light": "./light.js",
-    "protobufjs/minimal": "./minimal.js"
-  },
+  "browser": "index.js",
```

Comment:
I think the browser entry can simply be removed. As you pointed out, previous usage in the package.json was essentially wrong.

Reply:
Agreed, I will do that. Thanks!